### PR TITLE
Filter illegal chars in incoming soap requests

### PR DESCRIPTION
**Pull request by schuch**

### CODE REVIEW

File: connectors/csws/connector/src/main/java/org/apache/manifoldcf/csws/CswsSession.java [hunk 1]
```diff
@@ -174,13 +177,19 @@ public CswsSession(final String userName,
...
       this.searchServiceHandle = searchServiceService.getBasicHttpBindingSearchService();
+      installInvalidCharInterceptor(this.searchServiceHandle);
       ((BindingProvider)searchServiceHandle).getRequestContext().put(BindingProvider.ENDPOINT_ADDRESS_PROPERTY, searchServiceServiceURL);
     } else {
       this.searchServiceHandle = null;
     }
 
   }
 
+  private static void installInvalidCharInterceptor(Object handle) {
```
Comment (mil7) [inline connectors/csws/connector/src/main/java/org/apache/manifoldcf/csws/CswsSession.java]: Why did you add the functionality not for all handels?

Comment (schuch) [reply connectors/csws/connector/src/main/java/org/apache/manifoldcf/csws/CswsSession.java]: I expect no quality problems for the other entities / services.
And as we use regexp to replace the illegal chars there might be a performance impact.

Comment (mil7) [reply connectors/csws/connector/src/main/java/org/apache/manifoldcf/csws/CswsSession.java]: You are right.
